HousingRepairsOnlineApi/Controllers/RepairController.cs

```diff
@@ -16,18 +16,38 @@ public class RepairController : ControllerBase
         private readonly IAppointmentConfirmationSender appointmentConfirmationSender;
         private readonly IBookAppointmentUseCase bookAppointmentUseCase;
         private readonly IInternalEmailSender internalEmailSender;
+        private readonly IRetrieveRepairsUseCase retrieveRepairsUseCase;
+
 
         public RepairController(
             ISaveRepairRequestUseCase saveRepairRequestUseCase,
             IInternalEmailSender internalEmailSender,
             IAppointmentConfirmationSender appointmentConfirmationSender,
-            IBookAppointmentUseCase bookAppointmentUseCase
+            IBookAppointmentUseCase bookAppointmentUseCase,
+            IRetrieveRepairsUseCase retrieveRepairsUseCase
         )
         {
             this.saveRepairRequestUseCase = saveRepairRequestUseCase;
             this.internalEmailSender = internalEmailSender;
             this.appointmentConfirmationSender = appointmentConfirmationSender;
             this.bookAppointmentUseCase = bookAppointmentUseCase;
+            this.retrieveRepairsUseCase = retrieveRepairsUseCase;
+        }
+
+        [HttpGet]
+        [Route("PropertyRepairsCommunal")]
+        public async Task<IActionResult> PropertyRepairsCommunal([FromQuery] string propertyReference)
```

Comment:
I think this would read better as `CommunalPropertyRepairs` - not a required/requested changes, up to you to consider. (If you do change it, also change the Route attribute too).